pkg/template/controller/metrics.go

```diff
@@ -0,0 +1,71 @@
+package controller
+
+import (
+	"time"
+
+	templateapi "github.com/openshift/origin/pkg/template/apis/template"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/labels"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+func newTemplateInstancesTotal() *prometheus.GaugeVec {
+	return prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "TemplateInstanceController_TemplateInstances_total",
```

Comment:
I would suggest `openshift_template_instance_total`

Reply:
if we rename the controller, I'm supposing that templateinstance_controller_templateinstances_total might work, for example.